.yarnrc.yml

```diff
@@ -7,6 +7,17 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-deduplicate.cjs
     spec: "https://raw.githubusercontent.com/eps1lon/yarn-plugin-deduplicate/latest/bin/%40yarnpkg/plugin-deduplicate.js"
+  - path: .yarn/plugins/@yarnpkg/plugin-babel-version.cjs
```

Comment:
Out of curiosity, what if we offers `.yarn/plugins/@yarnpkg/plugin-babel-version.mjs` instead? @arcanis Does yarn 2 support `.mjs` plugins?

Reply:
Yarn's plugin bundler can only output CommonJS files.